README.md

```diff
@@ -251,6 +252,28 @@ vagrant ssh -c 'vault read consul/creds/admin-team'
 
 > :bulb: Tokens can be used to access UI (different access level depends on policy attached to the token)
 
+##### Vault PKI
+
+| default   | environment variable             |  value  |
+|:---------:|:---------------------------------|:-------:|
+|     x     | vault_pki                        |  true   |
+|           | vault_pki                        |  false  |
+
+[Vault PKI](https://www.vaultproject.io/docs/secrets/pki) will be enabled at `/pki`. A role called `default` is available to issue certificates.
```

Comment:
Where is `/pki` enabled? Does this means localhost:8200/pki?

If so maybe write that specificly.